src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -106,72 +106,28 @@ public class Result extends DomainObject {
     @Column(name = "example_result")
     private Boolean exampleResult;
 
+    // The following attributes are only used for Programming Exercises
+    @Column(name = "test_case_amount")
```

Comment:
Early comment, as it is relevant for the DB schema: There is a countable number of test cases. The attributes and columns should therefore be named with `count` instead of `amount`.